### PR TITLE
NDG tile

### PR DESCRIPTION
**Pull request by J0WI**
See #50

### CODE REVIEW

**Comment by MichaelKohler** [comment]
mh, now the details overlay is higher than before.. can you revert that change? I think it looks better if it just fits the content with a nice comfortable padding. otherwise it's perfect :)

**Comment by J0WI** [comment]
That was f2c153d
I need to specify a height, that you can scroll on mobile devices.

File: app/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ tiles:
     helpurl: https://support.mozilla.org/
     privacy: Value your privacy!
     mail: Contact
+    ndg: Nein zum Überwachungsstaat
```
Comment (freaktechnik) [inline app/Resources/translations/messages.en.yml]: "No to surveillance"?

File: app/Resources/public/less/index.less [hunk 2]
```diff
@@ -341,3 +354,35 @@ html[lang|="en"] .promo-small-landscape.privacy {
...
+.ndgbox a {
+    display: block;
+    padding: 1em;
+    margin: 2em;
+    font-size: 18px;
+    border-radius: 6px;
+    color: rgb(255, 255, 255);
+    background-color: rgb(39, 174, 96);
+    border-color: rgb(34, 153, 85);
+
```
Comment (freaktechnik) [inline app/Resources/public/less/index.less:370]: We can actually do without the `.flexbox` class with the following changes:
- Make text in the `.ndgbox` aligned to `center`
- Make `.ndgbox a` `inline-blocks`.

File: app/Resources/public/less/index.less [hunk 3]
```diff
@@ -216,6 +216,19 @@ html[lang|="en"] .promo-small-landscape.privacy {
     font-size: 1.3em;
 }
 
+.promo-large-portrait.ndg {
+    background-color: #27AE60;
+    background-image: url("/images/camera.svg");
```
Comment (freaktechnik) [inline app/Resources/public/less/index.less:221]: Can we somehow make text & icon have the same color (`fill: currentColor` or something along those lines)?

File: app/Resources/public/less/overlay.less [hunk 4]
```diff
@@ -27,7 +27,7 @@
     }
 }
 
-#close_overlay {
+.close-overlay {
```
Comment (freaktechnik) [inline app/Resources/public/less/overlay.less:30]: This broke the hover override for the link.

**Comment by MichaelKohler** [comment]
And of course @freaktechnik sees more than I do, nice!

For the height, it would be great if we could find a solution here. Now it looks quite empty.. :( maybe we could center it vertically as well?

**Comment by freaktechnik** [comment]
I don't think we should need to set a height in order for the container to be scrollable, it just needs an actual height (as in it can't have no height due to float or positioning shenanigans).

File: app/Resources/public/less/index.less (hunk 3 above)
Comment (J0WI) [inline app/Resources/public/less/index.less:221]: Nope, this doesn't work for cross documents.

File: app/Resources/public/less/index.less (hunk 2 above)
Comment (J0WI) [inline app/Resources/public/less/index.less:370]: Are there any advantages?

**Comment by J0WI** [comment]
I found a perfect way to handle the height problem using calc. Sadly, less isn't able to handle it. Any ideas?

**Comment by J0WI** [comment]
nvm, found a way to escape it. I hate CSS pre processors btw...

File: app/Resources/public/less/overlay.less (hunk 4 above)
Comment (J0WI) [inline app/Resources/public/less/overlay.less:30]: fixed in afe70b2

**Comment by J0WI** [comment]
Do we have any blockers left?

File: app/Resources/public/less/index.less (hunk 2 above)
Comment (freaktechnik) [inline app/Resources/public/less/index.less:370]: Way less CSS. Though we could probably shim the cross-browser stuff using a less thing (or maybe even from bedrock).

File: app/Resources/public/less/overlay.less [hunk 5]
```diff
@@ -22,12 +22,12 @@
             background: white;
             padding: 1em;
             height: 90%;
-            overflow: scroll;
+            overflow: auto;
```
Comment (freaktechnik) [inline app/Resources/public/less/overlay.less:25]: Does the marketplace overlay still work correctly with this change?

File: app/Resources/public/less/index.less (hunk 3 above)
Comment (J0WI) [inline app/Resources/public/less/index.less]: Yes, this just adds a scrollbar if necessary. E.g. the ugly horizontal scrollbar is now hidden.

File: app/Resources/public/less/overlay.less (hunk 5 above)
Comment (J0WI) [inline app/Resources/public/less/overlay.less:25]: Yes, this just adds a scrollbar if necessary. E.g. the ugly horizontal scrollbar is now hidden.

**Comment by J0WI** [comment]
Okay, I removed the fancy flexbox in favour of simple inline-blocks

File: app/Resources/public/less/overlay.less (hunk 5 above)
Comment (freaktechnik) [inline app/Resources/public/less/overlay.less:25]: I remember it not showing it on mobile sometimes, that's why I set it to scroll.

Comment (J0WI) [inline app/Resources/public/less/overlay.less:25]: WFM on Gecko

**Comment by J0WI** [comment]
Do we have now any blockers left? @freaktechnik @MichaelKohler 
I would like to merge this soon, because we have just a few weeks left!

**Comment by MichaelKohler** [comment]
Apart from my comments, no. Is the height fixed? Otherwise we're good to go from my side.

**Comment by J0WI** [comment]
yay, thx!

**Pull request merged**